src/api/submission/validations/validators/invert_obs.py

```
@@ -0,0 +1,239 @@
+
+    @validator_result
+    def __call__(self, collect_record, **kwargs):
+        observations = self.get_value(collect_record, self.observations_path) or []
+        transect = self.get_value(collect_record, self.transect_path) or {}
+
+        try:
+            width_id = transect.get("width")
+            len_surveyed = float(transect.get("len_surveyed") or 0)
+            width_val = float(InvertBeltTransectWidth.objects.get(pk=width_id).val)
+            area_m2 = width_val * len_surveyed
+        except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)